### PR TITLE
Clarify TLS key wiping

### PR DESCRIPTION
**Pull request by ravenac95**

### CODE REVIEW

**Comment by ravenac95** [comment]
@kostko is there actually a way for people to just regenerate the keys locally if they have their node keys all on their local machine?

**Comment by ptrus** [comment]
> is there actually a way for people to just regenerate the keys locally if they have their node keys all on their local machine?

`node identity init` probably (but that will probably generate other keys as well, so should discard those). Also not sure what's the benefit as the keys need to be on the node anyway?

**Comment by ravenac95** [comment]
> `node identity init` probably (but that will probably generate other keys as well, so should discard those). Also not sure what's the benefit as the keys need to be on the node anyway?

ya i wasn't sure if that command overwrote the other files. Oh this is mostly a curiosity.

File: src/operators/current-testnet-parameters.md [hunk 1]
```diff
@@ -61,7 +61,11 @@ Relevant changes to the configuration are detailed below.
    release](https://github.com/oasislabs/public-testnet-artifacts/releases/tag/2020-02-11)
    to the path `/serverdir/etc/genesis.json`.
 4. Delete `/serverdir/node/tls_identity.pem`.
+    * The format for the TLS keys has changed. This will be regenerated on
```
Comment (tjanez) [inline src/operators/current-testnet-parameters.md]: Could you just collapse 4. and 5. to Delete `/serverdir/node/tls_identity.pem` and `/serverdir/node/tls_identity_cert.pem`?

Comment (ravenac95) [reply src/operators/current-testnet-parameters.md]: AH shoot i just now saw this. I wanted to be extra explicit but you're right...

**Pull request merged**